app/src/main/java/org/sopt/and/ui/signin/SignInViewModel.kt

```diff
@@ -0,0 +1,52 @@
+package org.sopt.and.ui.signin
+
+import androidx.compose.runtime.State
+import androidx.compose.runtime.mutableStateOf
+import androidx.lifecycle.ViewModel
+import org.sopt.and.model.UserInfo
+
+class SignInViewModel : ViewModel() {
+
+    private val _userEnteredInfo = mutableStateOf(UserInfo())
+    val userEnteredInfo: State<UserInfo> = _userEnteredInfo
+
+    private val _snackbarMessage = mutableStateOf("")
+    val snackbarMessage: State<String> = _snackbarMessage
+
+    private val _showPassword = mutableStateOf(false)
+    val showPassword: State<Boolean> = _showPassword
```

Comment:
[이 리뷰](https://github.com/AND-SOPT-ANDROID/minseo-song/pull/4#discussion_r1826503397) 한번 살펴보시겠어요

Reply:
val이 아닌 var을 사용한다는 점에서 val이 가지는 이점이 사라지는게 아닌가 하는 생각에 디컴파일해서 보니 두 방법 모두 final로 정의 되어있어 동일한 역할을 한다는 사실을 확인했습니다.
제 코드에서는 아래와 같이 사용 가능할거 같습니다! 확실히 간결해 보이긴 한데 명확성이 이전 보다 떨어지진 않나? 하는 고민이 드는거 같습니다!
```suggestion
    var userEnteredInfo by mutableStateOf(UserInfo())
        private set
```

Reply:
어떤 부분에서 명확성이 떨어지는지 설명해주실 수 있을까요?